unittests/DependencyScan/Features.cpp

```diff
@@ -64,6 +64,6 @@ TEST_F(ScanTest, TestDoesNotHaveArgumentQuery) {
   bool hasOption;
   hasOption = optionSet.find("-clearly-not-a-compiler-flag") != optionSet.end();
   EXPECT_EQ(hasOption, false);
-  hasOption = optionSet.find("-emit-modul") != optionSet.end();
+  hasOption = optionSet.find("-emit-module") != optionSet.end();
```

Comment:
Wow, this has never triggered an error? Seems like we have insufficient test coverage :(

Reply:
I believe this is intentional, as it's followed by `EXPECT_EQ(hasOption, false);` and this test seems to be about arguments that don't exist. I assume it's meant to be very close to a real argument name, maybe to ensure nothing is doing a prefix match or something like that?

Reply:
@mikeash makes sense, I will revert the change then?

Reply:
Yes, please. Thanks!

Reply:
@mikeash fixed!